op_crates/file/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 
 [package]
 name = "deno_file"
-version = "0.1.0"
```

Comment:
We never released the 0.1.0 of this, but doesn't matter. @bartlomieju can you add us on crates.io?

Reply:
Done

Reply:
Thanks :-)